WARNING (2023-07-31 22:25:02,172): Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000021227EB86D0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0224_1823_B2.html
WARNING (2023-07-31 22:25:04,176): Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000021228131930>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0224_1823_B2.html
WARNING (2023-07-31 22:25:08,181): Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000212281306A0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0224_1823_B2.html
WARNING (2023-07-31 22:29:30,232): Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002122801E380>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0215_2100_B2F.html
WARNING (2023-07-31 22:29:32,247): Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000021228366740>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0215_2100_B2F.html
WARNING (2023-07-31 22:29:36,260): Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000021228365F60>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0215_2100_B2F.html
WARNING (2023-07-31 22:31:57,231): Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000212285CB460>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0210_1119_B2.html
WARNING (2023-07-31 22:31:59,247): Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000021228597A00>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0210_1119_B2.html
WARNING (2023-07-31 22:32:03,261): Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000021228595AB0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0210_1119_B2.html
WARNING (2023-07-31 22:33:23,647): Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002122827A6B0>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0207_1501_B2.html
WARNING (2023-07-31 22:37:47,202): Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000212283B1D80>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0201_1744_B1.html
WARNING (2023-07-31 22:37:55,232): Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000212283B3C40>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0201_1744_B1.html
WARNING (2023-07-31 22:38:11,293): Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000212283B0F10>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0201_1744_B1.html
WARNING (2023-07-31 22:38:25,236): Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000212283B0D90>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed')': /2023_Earthquake_Information/February/2023_0201_1744_B1.html
